src/runtime/entries/netlify-lambda.ts

```diff
@@ -30,9 +31,20 @@ export async function lambda(
     body: event.body, // TODO: handle event.isBase64Encoded
   });
 
+  const headers = normalizeOutgoingHeaders(r.headers);
+  // image buffers must be base64 encoded
+  if (Buffer.isBuffer(r.body) && headers["content-type"].startsWith("image/")) {
```

Comment:
this header check is hacky, looking for a better alternative

Reply:
If we trust mime.getType to return the correct type, this should be fine for images, as all images types should start with image. 
https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types.
However we should do this for all [Binary media](https://docs.aws.amazon.com/apigateway/latest/developerguide/lambda-proxy-binary-media.html)
As always with AWS, the list of supported media types isn't documented anywhere, but chatGPT gives me this:

image/png
image/jpeg
image/gif
application/octet-stream
application/pdf
application/x-zip-compressed
audio/mpeg
audio/mp4
video/mp4
application/zip
audio/webm
video/webm

Which sorts of make sense... But I think a cleaner way to handle all possible use cases without relying on the header would be to use https://www.npmjs.com/package/isbinaryfile